addon/components/validated-input.js

```diff
@@ -20,6 +20,8 @@ export default Ember.Component.extend({
 
   type: 'input',
 
+  // hint (provided by caller)
+
```

Comment:
Since we're not doing this anywhere else in this addon, I'd prefer to leave the comment out.